src/components/About/OurStory.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import '../../styles/About.css';
+import OpeningHeader from '../Home/OpeningHeader.js';
```

Comment:
What did you have in mind when you included `import OpeningHeader from '../Home/OpeningHeader.js';` in OurStory.js?